python-sdk/nuscenes/eval/detection/render.py

```diff
@@ -87,6 +87,9 @@ def visualize_sample(nusc: NuScenes,
         if box.score >= conf_th:
             box.render(ax, view=np.eye(4), colors=('b', 'b', 'b'), linewidth=1)
 
+    # Add legend
```

Comment:
Add a full-stop at the end of the sentence (this is in line with the style for `nuscenes-devkit`)